people-and-planet-ai/weather-forecasting/trainer/task.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Trains a Fully Convolutional Network to predict precipitation."""
+
+from __future__ import annotations
+
+from glob import glob
+import os
+from typing import Any as AnyType, Optional
+
+from datasets.arrow_dataset import Dataset
+from datasets.dataset_dict import DatasetDict
+import numpy as np
+import torch
+from transformers import (
```

Comment:
I didn't realize this was a HuggingFace model!

Reply:
Yeah, both `datasets` and `transformers` are HuggingFace, even if they don't "brand" their modules.